src/ExampleApp.cpp

```diff
@@ -53,12 +57,16 @@
 #include "BuildingSelectionExampleFactory.h"
 #include "RemoveMapLayerExampleFactory.h"
 #include "BillboardedSpriteExampleFactory.h"
+#include "VRCardboardExampleFactory.h"
+
+#include "Modules/VRDistortionModule/VRCardboardDeviceProfile.h"
+#include <android/log.h>
 
 namespace
 {
     Eegeo::Rendering::LoadingScreen* CreateLoadingScreen(const Eegeo::Rendering::ScreenProperties& screenProperties,
-                                                         const Eegeo::Modules::Core::RenderingModule& renderingModule,
```

Comment:
There's a large amount of whitespace change cruft through this PR, which makes it more difficult to find the real changes. Are the tab settings in your editor set correctly?